site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -42,6 +43,23 @@ This page describes the Admin UI for creating and configuring a Tenant.
 * <<Security>>
 * <<Advanced>>
 
+=== List of Tenants
+
+To display a list of tenants, navigate to [breadcrumb]#Tenants#.
+
+image::list-of-tenants.png[List of Tenants,width=1200,role=shadowed]
+
+[[create_duplicate_configuration]]
+Using the icons on this screen, you can:
+
+* icon:plus[role=ui-button green,type=fas] Create a new tenant 
+* icon:trash[role=ui-button red,type=fas] Delete the tenant configuration
+** You cannot delete the Default tenant.
```

Comment:
Please link to the limitations section here: https://fusionauth.io/docs/v1/tech/reference/limitations#default-configuration